src/main/java/org/jenkinsci/plugins/docker/traceability/DockerTraceabilityPlugin.java

```diff
@@ -55,10 +60,54 @@
      * @see DockerEventsAction#doRemoveContainer(org.kohsuke.stapler.StaplerRequest, org.kohsuke.stapler.StaplerResponse, java.lang.String)
      */
     public static final Permission DELETE = new Permission(PERMISSIONS,"Delete", Messages._DockerDeployment_Permissions_Delete_Permission_Description(), Jenkins.ADMINISTER, PermissionScope.JENKINS);
-       
+     
+    private DockerTraceabilityPluginConfiguration configuration;
+
+    /**
+     * Retrieves the plugin instance.
+     * @return {@link DockerTraceabilityPlugin}
+     * @throws IllegalStateException the plugin has not been loaded yet
+     */
+    public static @Nonnull DockerTraceabilityPlugin getInstance() {
+        Jenkins j = Jenkins.getInstance();
+        DockerTraceabilityPlugin plugin = j != null ? j.getPlugin(DockerTraceabilityPlugin.class) : null;
+        if (plugin == null) { // Fail horribly
+            // TODO: throw a graceful error
+            throw new IllegalStateException("Cannot get the plugin's instance. Jenkins or the plugin have not been initialized yet");
+        }
+        return plugin;
+    }
+    
+    @Override
+    protected void load() throws IOException {
+        super.load();
+        if (configuration == null) {     
+            configuration = DockerTraceabilityPluginConfiguration.getDefault();        
+            save();
+        }
+    }
+
+    void configure(DockerTraceabilityPluginConfiguration configuration) throws IOException {
+        this.configuration = configuration;
+        save();
+    }
+    
+    @Override
+    public void configure(StaplerRequest req, JSONObject formData) throws IOException, ServletException, Descriptor.FormException {
```

Comment:
I thought this needed to [return a boolean](http://javadoc.jenkins-ci.org/hudson/model/Descriptor.html#configure(org.kohsuke.stapler.StaplerRequest, net.sf.json.JSONObject))?